components/collective-page/CategoryHeader.js

```diff
@@ -99,23 +100,31 @@ const getCategoryData = (intl, collective, category) => {
         ),
       };
     case NAVBAR_CATEGORIES.CONTRIBUTE:
-      return {
-        img: contributeSectionHeaderIcon,
-        title: i18nNavbarCategory(intl, category),
-        subtitle: (
-          <FormattedMessage
-            id="CollectivePage.SectionContribute.Subtitle"
-            defaultMessage="Become a financial contributor."
-          />
-        ),
-        info: (
-          <FormattedMessage
-            id="CollectivePage.SectionContribute.info"
-            defaultMessage="Support {collectiveName} by contributing to them once, monthly, or yearly."
-            values={{ collectiveName: collective.name }}
-          />
-        ),
-      };
+      return isEventWithNoCustomContributions(collective) && !isAdmin
+        ? {
+            img: contributeSectionHeaderIcon,
+            title: intl.formatMessage({
+              id: 'CollectivePage.SectionContribute.Title.TicketsWithNoCustomContribution',
```

Comment:
We should re-use the existing `section.tickets.title`

Reply:
Ops, sorry, I always seems to miss this check. Corrected it. 👍🏽